common/lib/xmodule/xmodule/js/main_requirejs.js

```diff
@@ -0,0 +1,8 @@
+(function(requirejs) {
+    requirejs.config({
+        paths: {
+            "moment": "xmodule/include/common_static/js/vendor/moment.min"
```

Comment:
@cahrens 399 Jasmine tests are failing for xmodule when we remove this ```main_requirejs.js``` and use ```moment``` as dependancy in ```01_initialize.js```.